plenum/cli/cli.py

```diff
@@ -911,7 +911,7 @@ def newNode(self, nodeName: str):
                 self.bootstrapKey(self.activeWallet, node)
 
             for identifier, verkey in self.externalClientKeys.items():
-                node.clientAuthNr.addClient(identifier, verkey)
+                node.clientAuthNr.addIdr(identifier, verkey)
```

Comment:
What is Idr? Is it Identifier? I think we should use full name unless it's obvious. Please rename it to be more clear.